docs/source/installation.mdx

```diff
@@ -94,65 +87,69 @@ The location of this wrapper script is slightly different based on how you've in
 <a name="carthage-run-script" />
 <CarthageRunScriptPanel />
 
-## Build your target
+## 6. Build your target
 
-At this point, you can try building your target in Xcode. This will verify that the `schema.json` file can be found by the `apollo` script created above, and run the codegen.
+At this point, you can try building your target in Xcode. This verifies that the code generation build step can correctly locate your `schema.json` file.
 
 ### Troubleshooting
 
-If you get this error: 
+If you get this error:
 
-> Cannot find GraphQL schema file [...]
+> `Cannot find GraphQL schema file [...]`
 
-The script is not able to find your schema file - double check the path you've used.
+The script can't locate your `schema.json` file. Double check the path you've used.
```

Comment:
this should not be hard-coded to be schema.json since it can also be a) Named something else or b) A different file type.